hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java

```diff
@@ -385,7 +385,7 @@ public Set<DatanodeDetails> dnToFail() {
   }
   
   @Override
-  public void restartHddsDatanode(DatanodeDetails dn, boolean waitForDatanode)
+  public void restartHddsDatanode(DatanodeDetails dn, boolean Datanode)
```

Comment:
This looks like unintentional change.

Reply:
Changes have been addressed and committed.